content/en/technical-advisory-group/architectural-review-committee/ARC-001.en.md

```diff
@@ -7,4 +7,124 @@ draft: false
 lang: "en"
 ---
 
-TODO
+**Date:** March 26th, 2025
+**Location:** [Virtual/In-Person]  
+**Presenter:** William Hearn (Senior Cloud Architect, GC Cloud One)  
+**Attendees:** ARC Members
+**Objective:** Review the Aurora initiative and secure approval for its **strategic direction and technical architecture**.
+
+---
+
+## 1. Executive Summary
+
+### **Context**
+
+The **Aurora Initiative** is a **unified application hosting platform** designed to standardize **managed Kubernetes cloud offerings** across SSC. It integrates **both proprietary (OpenShift) and OSS (Cloud Native Kubernetes) environments**, drawing from operational experience in the **Cloud Native Platform (CNP)**.
+
+### **Why Now?**
+
+- Current departmental cloud-native strategies are **disjointed, costly, and redundant**.
+- Aurora provides a **governance framework** that reduces inefficiencies, promotes **multi-cloud standardization**, and aligns with SSC’s **GC Cloud One** strategy.
+- The initiative has **received broad buy-in** from multiple departments and aligns with similar government initiatives (BC Gov DevOps, Platform One, etc.).
+
+---
+
+## 2. Architecture Overview
+
+### **2.1 Technical Architecture**
```

Comment:
It might be good to emphasize that Aurora is a set of add-on components that bring a full-featured app platform to a Kubernetes cluster. So it uses the cloud Kubernetes offerings as a foundation and builds on-top of it.